paddle/fluid/operators/reshape_op.cc

```diff
@@ -226,7 +226,9 @@ class ReshapeKernel {
     }
 
     out->mutable_data(ctx.GetPlace(), in->type());
-    framework::TensorCopySync(*in, ctx.GetPlace(), out);
+    framework::TensorCopy(
+        *in, ctx.GetPlace(),
+        ctx.template device_context<platform::DeviceContext>(), out);
```

Comment:
@velconia why was this copy sync?

Reply:
This code is a copied from original implementation, but I guess Sync is NO need here.